### PR TITLE
Update README to clarify some of the steps

### PR DESCRIPTION
**Pull request by the-real-cphillips**
In doing our tech test, we realized that candidates might need more clarity around the tasks being asked.

Often they miss the EC2 Instance Count of Three, so updating to clarify, and a lot of times recently, they've tried to spin up VPC's and what not.

It can get hairy, so in the essence of transparency and clarity.

These updates were born!

Also just reworded some of the other items to hopefully clear up the confusion we've seen going through this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,59 @@
...
-      * `us-west-2` region
-    * Each instance should have an 100G EBS Volume attached
-    * Each instance should have separate `Name` tags in AWS
+What to build:
+    * Total of three (3) EC2 Instances with the following:
+        * Instance Type: `m5.large`
+        * AMI: `ami-094125af156557ca2`
+        * Region: `us-west-2`
+    * Each instance should have it's own 100G EBS Volume attached
+    * Each instance should have a separate `Name` tags in AWS
```
Comment (s33dunda) [inline README.md]:
```suggestion
    * Each instance should be tagged with a unique `Name`
```

**Review by s33dunda — COMMENTED** [review]
We should say the name tag should be unique. Phrasing can be however

**Comment by jbanks212** [comment]
We should also clarify the EBS volume should not be root but an additional volume. No?

File: README.md [hunk 2]
```diff
@@ -1,50 +1,59 @@
 # SRE Technical Test
-Infrastructure is best represented as code and provisioning of resources should be automated as much as possible. Here at 2U, [Terraform](terraform.io) is our tool of choice. Below you will find 2 tasks that will help demonstrate your apptitude for these required tools
+Infrastructure is best represented as code and provisioning of resources should be automated as much as possible. 
+
+Here at 2U, [Terraform](terraform.io) is our tool of choice. Below you will find 2 tasks that will help demonstrate your apptitude for these required tools
 
 The first task, under `Infrustructure`, write some simple `terraform`. If you have any questions regarding how to work within this space, please ask your interviewers.
```
Comment (jbanks212) [inline README.md]: Infrastructure spelling correction on ln 6

**Review by the-real-cphillips — COMMENTED** [review]
yes

**Pull request merged**